source/s3_copy_object.c

```diff
@@ -60,7 +60,7 @@ static int s_s3_copy_object_request_type(struct aws_s3_request *request) {
 
 static struct aws_s3_meta_request_vtable s_s3_copy_object_vtable = {
     .update = s_s3_copy_object_update,
-    .send_request_finish = aws_s3_meta_request_send_request_finish_handle_async_error,
+    .send_request_finish = aws_s3_meta_request_send_request_finish_default,
```

Comment:
Every vtable.send_request_finish needs to call `aws_s3_meta_request_send_request_finish_default()` now, right?

If so, let's NOT have that be a vtable function we customize for the GET/PUT/DEFAULT/COPY vtables

Is any request-type doing things differently in there? If not, just remove the vtable function altogether, and we don't need to stress about how it's confusing with the other similarly named vtable function.

Reply:
ok, that explains it

How about on_request_complete_pre_retry / on_request_complete_post_retry?

Reply:
whatever